crates/bevy_ui/src/render/mod.rs

```diff
@@ -144,6 +146,25 @@ pub fn build_ui_render(app: &mut App) {
     app.add_plugins(UiTextureSlicerPlugin);
 }
 
+#[derive(Debug, Resource, Default, PartialEq, Eq, Clone)]
+pub enum ExtractedUiAntialias {
```

Comment:
Do we need this, we could just use the same `UiAntiAlias` type for the resource in the `RenderWorld`?

Reply:
Or newtype it

Reply:
My intuition is that we should be able to just use `ExtractResourcePlugin`, no?

Reply:
Yep that sounds even better.

Reply:
Thanks for the tip on `ExtractResourcePlugin`, that's really useful!